app/GRZMAK/GRZMAK_task2.c

```diff
@@ -0,0 +1,81 @@
+/**
+* Responsible: Grzegorz Makowski
+*
+*
+*History Log:
+* 2017.12.06 GRZMAK initial version
+*
+**/
+
+
+#include "GRZMAK_task2.h"
+
+
+
+eErr_t GRZMAK_setLampBit( eLamp_t bit, int* reg)
+{
+	if(bit < LAMP_START || bit >= LAMP_COUNT)
+	{
+		return E_INVALID_ARG_1;
```

Comment:
Please use single review per function (for example you can create result local variable and return it ea the end)